debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/rest/DebeziumMongoDbConnectorResource.java

```diff
@@ -41,6 +59,48 @@ public Connector getConnector() {
         return new MongoDbConnector();
     }
 
+    protected MongoDbConnection primary(MongoDbTaskContext context) throws DebeziumException {
```

Comment:
This is rather misleading name... primary is a term used for the leading node in mongo RS. This method seems to be creating our internal representation of the whole RS

Reply:
I am aware of what the term is used for and it is not exclusive for Mongo DB, is it?
I just copied this code from the UI backend (and maybe from legacy Mongo DB connector code?!). I have no idea what you want to change. Feel free to add a commit with better code or setup a call to explain this in detail.

Reply:
We are working in context of MongoDB so the interpretation is in the context of MongoDB :). 

And yes, I suspected this might be from legacy code -- we used to have this all over the place. The reason was that back then we were indeed connecting directly to RS primary so the name made sense back then. What about naming this "connection" or something simmilar

Reply:
I re-added enforcing now.